app/models/participant_declaration.rb

```diff
@@ -5,7 +5,7 @@ class ParticipantDeclaration < ApplicationRecord
   belongs_to :cpd_lead_provider
   belongs_to :user
 
-  validates :course_identifier, inclusion: { in: :valid_courses }
+  validates :course_identifier, inclusion: { in: :valid_courses, message: "The property '#/course_identifier' must be an available course to '#/participant_id'" }
```

Comment:
Should we allow this to be missing here, since the presence check following this should take precedence.

Reply:
I would not want it to be missed. I am happy to look at re-ordering it (and adding more meaningful tests) in a future PR.

Reply:
Did you mention that as part of https://dfedigital.atlassian.net/browse/CPDTP-362 you would like to have a look at how valuable our errors are?